tests/integration/src/test/scala/mesosphere/marathon/integration/NodeDrainingIntegrationTest.scala

```diff
@@ -0,0 +1,63 @@
+package mesosphere.marathon
+package integration
+
+import akka.stream.scaladsl.Sink
+import mesosphere.AkkaIntegrationTest
+import mesosphere.marathon.integration.setup.EmbeddedMarathonTest
+import mesosphere.marathon.raml.App
+import mesosphere.marathon.state.PathId
+
+class NodeDrainingIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest {
+
+  val pathId = PathId("/back-the-world-off")
```

Comment:
```suggestion
  val pathId = AbsolutePathId("/back-the-world-off")
```

We might just get started only using this one ;)